common/elasticsearch/esql/globals.go

```diff
@@ -85,11 +83,3 @@ const (
 	TieBreakerOrder      = "desc"
 	StartTimeOrder       = "desc"
 )
-
-// DEBUG usage
-//nolint
-func print(v interface{}) {
-	fmt.Println("==============")
-	fmt.Println(v)
-	fmt.Println("==============")
-}
```

Comment:
honestly I wonder if we can find a linter to just ban `fmt.Print*` across the board.  it's basically always better-served by some injected thing, like a logger

Reply:
Quickly searched for such usages. Majority of the comes in CLI or in tests. Those are probably valid uses.